components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -144,30 +149,53 @@ export default function () {
-                                        <button onClick={() => setIsStartWorkspaceModalVisible(true)}>
-                                            New Workspace{" "}
-                                            <span className="opacity-60 hidden md:inline">
-                                                {StartWorkspaceModalKeyBinding}
+                    <>
+                        {!hasPreferredIde && (
+                            <div className="px-6 py-10 flex flex-col text-gray-400 border-t border-gray-200 dark:border-gray-800">
+                                <div className="flex flex-col items-center justify-center mx-auto">
+                                    <>
+                                        <h3 className="text-center pb-3">Choose your Default IDE</h3>
+                                        <p className="text-base text-gray-500 dark:text-gray-400">
+                                            Gitpod is using Code as the default IDE. You can choose a different IDE or
+                                            use a custom IDE image.
+                                        </p>
+                                        <SelectIDE showLatest={false} />
+                                    </>
+                                </div>
+                            </div>
+                        )}
+                        {hasPreferredIde && (
+                            <div className="app-container flex flex-col space-y-2">
+                                <div className="px-6 py-3 flex flex-col text-gray-400 border-t border-gray-200 dark:border-gray-800">
+                                    <div className="flex flex-col items-center justify-center h-96 w-96 mx-auto">
+                                        <>
+                                            <h3 className="text-center pb-3 text-gray-500 dark:text-gray-400">
+                                                No Workspaces
+                                            </h3>
+                                            <div className="text-center pb-6 text-gray-500">
+                                                Prefix any Git repository URL with {window.location.host}/# or create a
+                                                new workspace for a recently used project.{" "}
+                                                <a
+                                                    className="gp-link"
+                                                    href="https://www.gitpod.io/docs/getting-started/"
+                                                >
+                                                    Learn more
+                                                </a>
+                                            </div>
+                                            <span>
+                                                <button onClick={() => setIsStartWorkspaceModalVisible(true)}>
+                                                    New Workspace{" "}
```

Comment:
We also need to provide `New Workpsace` button here in some way if `!hasPerferredIde`

Reply:
the user can skip in this case, and the regular flow for opening new workspaces will be there